### PR TITLE
First Capstone Project 💪🤓

### PR DESCRIPTION
**Pull request by hsztan**
First Capstone Project Using HTML, CSS, and JavaScript. 📝.
I created a main and about page 📑 which share the same CSS and Javascript files and add dynamic content to speakers 🗣 from an object .js file.
The mobile 📱 version uses a dynamic menu bar that pops up 🍾 when clicked, and to see more speakers, a "see more" a user may click a "see more" button 🖱.

Video description --> [view here](https://drive.google.com/file/d/1LgVQ9K23jV62zAwaEJQUYVAbKmcxsfzw/view?usp=sharing)

Updated Video description (shortened version) --> [view here](https://www.loom.com/share/cfc6acbb69664a60b7ff7d262627bee5)

### CODE REVIEW

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @hsztan, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight

- your design was really sleek and well crafted, you'd be a great frontend engineer
- your code organization was really good.

## Required Changes ♻️

- [x] Following the project requirements, you should record a video presenting the features of the project. Check out the requirements below 👇:
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). Don't change the branch name now, just try to correct this behavior in the future.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hsztan** [comment]
Will do. Thanks for the feedback! 🙂

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @hsztan, 👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight ✔️ 
As the previous code reviewer said, You will become a great front-end developer, I admire your UI design
Excellent communication with previous code reviewer 😇 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._


- [x] As per requirement, the Video length should be a maximum of 5 minutes length the reason being is that later on when I am a recruiter or someone going through your GitHub I can just view it real quick without having to spend too much time, right?. My suggestion to make the video shorter is to practice what you want to say and try to go through it a bit faster in the video without having to think too much of what we want to say in the video. :shipit: 
- [x] [OPTIONAL] The video quality was not doing you a favor on showing the cool animations if you can get a better internet/desk device to record this video It would make your UI design skills more appealing 😇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hsztan** [comment]
Understood. Will make a shorter video 🎥. Appreciate the feeback 🙂

File: about.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    <footer>
+      <div class="logo">
+        <img class="logo-img" src="./images/FIAC-logo-2X.png" alt="FIAC logo" />
+      </div>
+      <div class="copy">
+        <p>
+          2022 First Interplanetary Arrival Conference. Some Rights Reserved.
+        </p>
+      </div>
+    </footer>
```
Comment (MrBrN197) [inline about.html:121]: - [ ] [OPTIONAL] Hi, Well done on the design of your website. I'd like to point out that the desktop version of the about page should have a dark background according to the Figma design, I can see on yours you have a White background

File: about.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      </div>
+    </section>
+    <section class="organizers">
+      <h2>Organizers</h2>
+      <hr class="organizers-hr" />
+      <div class="organizers-wrapper">
+        <img
+          src="./images/sponsor-logos/china_national_space_administration_logo.png"
+          alt=""
+        />
```
Comment (MrBrN197) [inline about.html:100]: - [ ] [OPTIONAL] Please try to include an alternate text in every `img` tag to help with accessiblity.

File: index.html [hunk 3]
```diff
@@ -5,9 +5,143 @@
...
+      </div>
+    </section>
+    <section class="organizers" id="organizers">
+      <h2>Organizers</h2>
+      <hr class="organizers-hr" />
+      <div class="organizers-wrapper">
+        <img
+          src="./images/sponsor-logos/china_national_space_administration_logo.png"
+          alt=""
+        />
```
Comment (MrBrN197) [inline index.html:122]: - [ ] [OPTIONAL] Please try to include an alternate text in every `img` tag to help with accessiblity.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @hsztan  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**